docs/guides/fungible-token.md

```diff
@@ -0,0 +1,590 @@
+---
+title: How to Create a Fungible Token on Flow
+sidebar_label: Create a Fungible Token
+description: Guide to creating a fungible token on Flow with the Flow CLI and Cadence.
+sidebar_position: 4
+------
+
+This article will guide you through the steps to create, deploy, mint, and transfer fungible tokens on Flow.
+
+## What are Fungible Tokens?
+
+Fungible tokens are digital assets that are interchangeable and indistinguishable with other tokens of the same type. This means that each token is identical in specification to every other token in circulation. Think of them like traditional money; every dollar bill has the same value as every other dollar bill. Fungible tokens play a crucial role in web3 ecosystems, serving as both a means of payment and an incentive for network participation. They can take on various roles including currencies, structured financial instruments, shares of index funds, and even voting rights in decentralized autonomous organizations.
+
+## Vaults on Flow
+
+On the Flow blockchain, fungible tokens are stored in structures called vaults. Think of a vault as a digital piggy bank. When you transfer tokens from one vault to another:
```

Comment:
I would recommend linking to the cadence tutorial or something that goes into a little more detail about how resources are used to represent vaults on Flow for tokens

Reply:
Feel free to add a PR for this update @joshuahannan. Would be much appreciated!